aioasuswrt/asuswrt.py

```diff
@@ -67,8 +67,6 @@
         if line:
             match = regex.search(line)
             if not match:
-                _LOGGER.debug("Could not parse row: %s", line)
-                _LOGGER.debug(type(line))
```

Comment:
Did you mean to get rid of the error message?

Reply:
No thank you for heads up, its back now